app/views/authorized_representative/edit.html.erb

```diff
@@ -17,12 +17,16 @@
       builder: MbFormBuilder,
       url: current_path,
       method: :put do |f| %>
-    <%= f.mb_radio_set(
-      :authorized_representative,
-      "Would you like to designate someone else to be your authorized representative?",
-      [{ value: :true, label: "Yes" }, { value: :false, label: "No" }],
-      layout: "inline") %>
-    <%= f.mb_input_field :authorized_representative_name, "What is their full legal name?" %>
+
+    <div class="form-group">
+      <%= f.mb_radio_set(
+        :authorized_representative,
+        "Would you like to designate someone else to be your authorized representative?",
+        [{ value: :true, label: "Yes" }, { value: :false, label: "No" }],
```

Comment:
If we change the symbols to booleans (`:true:` to `true`) and (`:false` to `false`), would that address the issue of checking for the string `"false"` in the step?

Reply:
the controller params will always be strings, as far as I an tell.